server/app/auth/oidc/applicant/IdcsProfileAdapter.java

```diff
@@ -31,83 +25,42 @@
  * This class takes an existing CiviForm profile and augments it with the information from an IDCS
  * profile.
  */
-public class IdcsProfileAdapter extends OidcProfileAdapter {
+public class IdcsProfileAdapter extends OidcApplicantProfileAdapter {
```

Comment:
Not blocking: If we don't want or expect subclasses, mark the class as final to start. We can always open it up to subclassing later.